meta-phosphor/recipes-phosphor/interfaces/bmcweb_git.bb

```diff
@@ -24,6 +24,7 @@ SRC_URI += " \
 "
```

Comment:
Please provide a brief reason on why there is not a link to upstream for this? Still being discussed in upstream?